packages/protocol/contracts/governance/Validators.sol

```diff
@@ -239,6 +248,34 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
     return true;
   }
 
+  /**
+   * @notice Sets the election threshold.
+   * @param threshold Election threshold as unwrapped Fraction.
+   * @return True upon success.
+   */
+  function setElectionThreshold(uint256 threshold)
+    public
+    onlyOwner
+    returns (bool)
+  {
+    electionThreshold = FixidityLib.wrap(threshold);
+    require(
+      FixidityLib.lt(electionThreshold, FixidityLib.fixed1()),
+      "Election threshold must be lower than 100%"
+    );
+    emit ElectionThresholdSet(threshold);
+    return true;
+  }
+
+  /**
+   * @notice Gets the election threshold.
+   * @return Threshold value as unwrapped fraction.
+   */
+  function getElectionThreshold() external view returns (uint256) {
+    return FixidityLib.unwrap(electionThreshold);
```

Comment:
Style nit: you can do `electionThreshold.unwrap()` thanks to the `using for` declaration at the top of the contract.